diff --git mDNSWindows/SystemService/mDNSResponder.vcxproj mDNSWindows/SystemService/mDNSResponder.vcxproj
index 47f2708..bbdefb5 100644
--- mDNSWindows/SystemService/mDNSResponder.vcxproj
+++ mDNSWindows/SystemService/mDNSResponder.vcxproj
@@ -307,7 +307,6 @@
     <ClCompile Include="..\PosixCompat.c" />
     <ClCompile Include="Firewall.cpp" />
     <ClCompile Include="..\..\mDNSShared\GenLinkedList.c" />
-    <ClCompile Include="..\..\mDNSMacOSX\LegacyNATTraversal.c" />
     <ClCompile Include="main.c" />
     <ClCompile Include="..\..\mDNSCore\mDNS.c" />
     <ClCompile Include="..\..\mDNSShared\mDNSDebug.c" />
diff --git mDNSWindows/SystemService/mDNSResponder.vcxproj.filters mDNSWindows/SystemService/mDNSResponder.vcxproj.filters
index 50d1e14..4edda5d 100644
--- mDNSWindows/SystemService/mDNSResponder.vcxproj.filters
+++ mDNSWindows/SystemService/mDNSResponder.vcxproj.filters
@@ -33,9 +33,6 @@
     <ClCompile Include="..\..\mDNSShared\GenLinkedList.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\mDNSMacOSX\LegacyNATTraversal.c">
-      <Filter>Source Files</Filter>
-    </ClCompile>
     <ClCompile Include="main.c">
       <Filter>Source Files</Filter>
     </ClCompile>
diff --git mDNSWindows/SystemService/mDNSResponderDLL.vcxproj mDNSWindows/SystemService/mDNSResponderDLL.vcxproj
index 1912b7f..387a473 100644
--- mDNSWindows/SystemService/mDNSResponderDLL.vcxproj
+++ mDNSWindows/SystemService/mDNSResponderDLL.vcxproj
@@ -298,7 +298,6 @@
     <ClCompile Include="..\..\mDNSCore\DNSDigest.c" />
     <ClCompile Include="..\..\mDNSCore\mDNS.c" />
     <ClCompile Include="..\..\mDNSCore\uDNS.c" />
-    <ClCompile Include="..\..\mDNSMacOSX\LegacyNATTraversal.c" />
     <ClCompile Include="..\..\mDNSShared\ClientRequests.c" />
     <ClCompile Include="..\..\mDNSShared\DebugServices.c" />
     <ClCompile Include="..\..\mDNSShared\dnssd_ipc.c" />
diff --git mDNSWindows/SystemService/mDNSResponderDLL.vcxproj.filters mDNSWindows/SystemService/mDNSResponderDLL.vcxproj.filters
index 3a301e0..1287ef9 100644
--- mDNSWindows/SystemService/mDNSResponderDLL.vcxproj.filters
+++ mDNSWindows/SystemService/mDNSResponderDLL.vcxproj.filters
@@ -98,9 +98,6 @@
     <ClCompile Include="..\..\mDNSShared\GenLinkedList.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\mDNSMacOSX\LegacyNATTraversal.c">
-      <Filter>Source Files</Filter>
-    </ClCompile>
     <ClCompile Include="..\..\mDNSCore\mDNS.c">
       <Filter>Source Files</Filter>
     </ClCompile>
